nixos/modules/services/desktop-managers/lomiri.nix

```diff
@@ -38,6 +38,7 @@ in {
       ]);
     };
 
+    hardware.pulseaudio.enable = lib.mkDefault true;
```

Comment:
You are sure we want to use legacy pulseaudio by default and pipewire?

Reply:
[UBports is using PA still](https://docs.ubports.com/en/latest/porting/configure_test_fix/Sound.html), as does Debian with their Lomiri packaging. I haven't had any issues with it so far, so I see no urgency to push for PW right now.